statick_tool/rsc/config.yaml

```diff
@@ -169,6 +184,9 @@ levels:
                        line-length: disable}}'"
 
   tex:
+    reporting:
+      print_to_console:
+      write_jenkins_warnings_ng:
```

Comment:
I don't think we want this plugin enabled for this level. No jobs I know of use this level with Jenkins.